data/config-schema/src/main/java/net/corda/schema/configuration/StateManagerConfig.java

```diff
@@ -0,0 +1,30 @@
+package net.corda.schema.configuration;
+
+/**
+ * Configuration keys to access public parts of the configuration under the {@code corda.stateManager} key.
+ */
+public final class StateManagerConfig {
+    private StateManagerConfig() {
+    }
+
+    public static final String STATE_MANAGER = "stateManager";
+
+    public static final String TYPE = "type";
+
+    // Database Configuration Values
+    public static final class Database {
+        public static final String DB_PROPERTIES = "database";
```

Comment:
Yeah that's better, so when users pass config to the stateManagerFactory.create() the configs don't need prefixed with a "stateManager" 👍

Reply:
Yeah: https://github.com/corda/corda-runtime-os/pull/4780/files#diff-0cbc84dd880a3bbc7bd717605c94972718d589a610ec6e7da73de771975944ecR43